packages/core/components/Puck/components/Sidebar/styles.module.css

```
@@ -0,0 +1,35 @@
+.Sidebar {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  overflow-y: auto;
+}
+
+.Sidebar--left {
+  background: var(--puck-color-grey-12);
+  border-inline-end: 1px solid var(--puck-color-grey-09);
+  grid-area: left;
+}
+
+.Sidebar--right {
+  background: var(--puck-color-white);
+  border-inline-start: 1px solid var(--puck-color-grey-09);
+  grid-area: right;
+}
+
+.ResizeHandleContainer {
```

Puck uses SUITCSS format, so this should be

```suggestion
.Sidebar-resizeHandle {
```